### PR TITLE
Clean w3c validator notices for role attributes

### PR DESCRIPTION
**Pull request by wroxbox**
Fixing some issues for w3c validator

w3c-validator-role-main

### CODE REVIEW

**Comment by tjwelde** [comment]
Hello @wroxbox,
thank you very much for contributing with your PR.
Unfortunately, we won't remove the role attribute from main, as long as it is default Drupal behaviour to have it.

**Comment by tjwelde** [comment]
I think it is there for compatibility reasons, because many blind people use screenreaders, which have to be used with old browsers, who only understand the attribute and not the HTML5 element.

**Comment by tjwelde** [comment]
Why do you want to remove the id?

File: templates/system/html.html.twig [hunk 1]
```diff
@@ -8,7 +8,6 @@ db_offline ? 'db-offline',
 <!DOCTYPE html>
 <html{{ html_attributes }}>
     <head>
-        <meta charset="utf-8">
```
Comment (tjwelde) [inline templates/system/html.html.twig:11]: Why are you removing the charset here?

Comment (wroxbox) [reply templates/system/html.html.twig:11]: It is double declared later :)

**Comment by saviomuc** [comment]
Closed du to inactivity

**Pull request closed without merging**